cherrypy/_cpcompat.py

```diff
@@ -332,3 +332,13 @@ def _args_from_interpreter_flags():
             args.append('-W' + opt)
 
         return args
+
+if six.PY2:
+    from cgi import escape as escape_html
```

Comment:
you **must not** fool user with another function name. especially when it's behavior differs. escape_html would be expected to have `quote=True`, whereas you break this expectation.